server/settings/publish_plugins.py

```diff
@@ -797,8 +797,8 @@ class PublishPuginsModel(BaseSettingsModel):
         default_factory=ValidateBaseModel,
         title="Validate Version"
     )
-    ValidateContainers: ValidateContainersModel = SettingsField(
-        default_factory=ValidateContainersModel,
+    ValidateOutdatedContainers: ValidateOutdatedContainersModel = SettingsField(
+        default_factory=ValidateOutdatedContainersModel,
         title="Validate Containers"
```

Comment:
Haha - of all the changes I actually mentioned in the other PR I really meant the **title** to be updated since that's what the user will see in the settings directly. 

Sorry, but the PR is already merged. But can we change the settings **title** to `Validate Outdated Containers`?

```python
        title="Validate Outdated Containers"
```